.github/scripts/keptnevaluationprovider_migrator.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+getNamespaces() {
+  out=$(kubectl get ns -o name)
+  for n in $out
+  do
+    nn=${n#"namespace/"}
+    namespaces+=("$nn")
+  done
+}
+
+getProvidersOfNamespace() {
+  out=$(kubectl get keptnevaluationproviders -o name -n "$1")
+  for n in $out
+  do
+    nn=${n#"keptnevaluationprovider.lifecycle.keptn.sh/"}
+    providers+=("$nn -n $1")
+  done
+}
+
+echo -e "------------------------------\n"
+echo -e "Migrating manifests.\n"
+echo -e "------------------------------\n"
+
+declare -a namespaces
+declare -a providers
+
+DATE=$(date +%s)
+MANIFESTS_FILE="manifests-$DATE.yaml"
+
+getNamespaces
+
+for n in "${namespaces[@]}"
+do
+  getProvidersOfNamespace "$n"
+done
+
+for n in "${providers[@]}"
+do
+  echo "---" >> $MANIFESTS_FILE
+  kubectl get keptnevaluationproviders $n -oyaml >> $MANIFESTS_FILE
+done
+
+sed -i 's/lifecycle\.keptn\.sh\/v1alpha2/metrics\.keptn\.sh\/v1alpha2/g' $MANIFESTS_FILE
```

Comment:
you could use different delimiters here to make the command easier to read:
```suggestion
sed -i 's#lifecycle\.keptn\.sh/v1alpha2#metrics\.keptn\.sh/v1alpha2#g' $MANIFESTS_FILE
```